app/sidekiq/publish_translation_job.rb

```diff
@@ -0,0 +1,13 @@
+class PublishTranslationJob
+  include Sidekiq::Job
+
+  def perform(id)
+    translation = Translation.find(id)
+    begin
+      translation.push_published_to_s3
+      translation.update!(is_published: true)
+    rescue => e
+      translation.errors.add(:error, e)
```

Comment:
hmm, are these `errors` stored in the database to where they can be present in the serializer on subsequent requests?

If not I'm fine with this being a simple text field on the `Translation` model that tracks publishing errors